providers/configcat/README.md

```diff
@@ -0,0 +1,52 @@
+# Unofficial ConfigCat OpenFeature Provider for Java
+
+[ConfigCat](https://configcat.com/) OpenFeature Provider can provide usage for ConfigCat via OpenFeature Java SDK.
+
+## Installation
+
+<!-- x-release-please-start-version -->
+
+```xml
+
+<dependency>
+    <groupId>dev.openfeature.contrib.providers</groupId>
+    <artifactId>configcat</artifactId>
+    <version>0.0.1</version>
+</dependency>
+```
+
+<!-- x-release-please-end-version -->
+
+## Usage
+ConfigCat OpenFeature Provider is using ConfigCat Java SDK.
+
+### Usage Example
+
+```
+ConfigCatProviderConfig configCatProviderConfig = ConfigCatProviderConfig.builder().sdkKey(sdkKey).build();
+configCatProvider = new ConfigCatProvider(configCatProviderConfig);
+OpenFeatureAPI.getInstance().setProviderAndWait(configCatProvider);
+boolean featureEnabled = client.getBooleanValue(FLAG_NAME, false);
+
+MutableContext evaluationContext = new MutableContext();
+evaluationContext.setTargetingKey("csp@matching.com");
+evaluationContext.add("Email", "a@b.com");
+evaluationContext.add("Country", "someCountry");
+featureEnabled = client.getBooleanValue(USERS_FLAG_NAME, false, evaluationContext);
```

Comment:
I'd prefer an immutable context example, but either is fine so up to you.

Reply:
I think ImmmutableContext only accept Value type objects?